runtime/autoload/man.vim

```diff
@@ -118,15 +118,12 @@ endfunction
 " Expects a string like 'access' or 'access(2)'.
 function s:parse_page_and_section(sect, str) abort
   try
-    let save_isk = &iskeyword
-    setlocal iskeyword-=(,)
-    let page = substitute(a:str, '(*\(\k\+\).*', '\1', '')
-    let sect = substitute(a:str, '\(\k\+\)(\([^()]*\)).*', '\2', '')
+    let page = substitute(a:str, '\([-.[:alnum:]_]\+\).*', '\1', '')
+    let sect = substitute(a:str, '\([-.[:alnum:]_]\+\)(\([^()]*\)).*', '\2', '')
     if sect == page || -1 == match(sect, '^[0-9 ]\+$')
```

Comment:
@noahfrederick :+1: So `man#get_page` will be less clever and will not do `expand('<cword>')` internally. Totally in favor of th.at